model/utils.go

```diff
@@ -213,10 +213,16 @@ func IsValidChannelIdentifier(s string) bool {
 	return true
 }
 
+var validAlphaNumUnderscore = regexp.MustCompile(`^[a-z0-9]+([a-z\-\_0-9]+|(__)?)[a-z0-9]+$`)
 var validAlphaNum = regexp.MustCompile(`^[a-z0-9]+([a-z\-0-9]+|(__)?)[a-z0-9]+$`)
 
-func IsValidAlphaNum(s string) bool {
-	match := validAlphaNum.MatchString(s)
+func IsValidAlphaNum(s string, underscores bool) bool {
```

Comment:
Super minor but I think the parameter would be better named 'allowUnderscores' 0/5 on whether you change this or not, since it's not a big deal